packages/wrangler/src/bundle.ts

```diff
@@ -92,14 +92,37 @@ export async function bundleWorker(
 		rules,
 	});
 
+	// In dev, we want to patch `fetch()` with a special version that looks
+	// for bad usages and can warn the user about them; so we inject
+	// `checked-fetch.js` to do so. However, with yarn 3 style pnp,
+	// we need to extract that file to an accessbile place before injecting
+	// it in, hence this code here.
+	let checkedFetchFileToInject: string | undefined;
+	if (checkFetch) {
+		fs.mkdirSync(path.join(destination, "--temp-wrangler-files--"), {
+			recursive: true,
+		});
+		checkedFetchFileToInject = path.join(
+			destination,
+			"--temp-wrangler-files--",
+			"checked-fetch.js"
+		);
```

Comment:
This line has no side-effect, so you could just move it to where the `checkedFetchFileToInject` variable is declared, avoiding the potential `undefined`.

Reply:
oh that's good. doing so, thanks!